### DISCUSSION TITLE
Customizing Pre-defined maps

### DISCUSSION THREAD

**Discussion by EricPolizzy** [carla-simulator/carla#4372] [General]
I was wondering whether there's a possibility that I can make a few changes to defined Towns?I know Layered maps could work, but it removes all the objects. However, I'm more interested in relocating a few objects, such as removing and rotating a few trees.  Thank you in advance!

**Comment by corkyw10** [comment]
Hi @EricPolizzy, I think you should be able to do this in the UE editor just fine. Are you facing as problems when you try it?
